src/pages/advance/clerk-idp.mdx

```diff
@@ -0,0 +1,69 @@
+import { Callout } from "nextra-theme-docs";
+import { Images } from "@/components/Images";
+
+# Use Clerk as an OAuth 2.0 Identity Provider 
+
+Learn how to use Clerk as an Identity provider to facilitate Single Sign-On (SSO) with other applications that support the OAuth2 protocol. 
+
+<Images
+  width={496}
+  height={564}
+  src="/images/advance/clerk-idp-flow.svg"
+  alt="Clerk SSO IDP Flow"
+/>
+
+
+## Step 1: Retrieve SSO Provider Callback URL
```

Comment:
I left a note about this in the slack thread, but I think the term `SSO provider` is misused here. Maybe we can align on a phrase in slack.

Reply:
You're right, i think Client probably makes the most sense in the OAuth context as well. So i'll change that.